spec/features/credit_cards_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature "CreditCardsControllers", type: :feature do
```

Comment:
since it's a feature test, we are testing the credit cards as a whole, not only the CreditCardsController. We usually use something like `RSpec.feature 'credit_cards', type: :feature do`

The same applies to the filename